src/ln/channelmonitor.rs

```diff
@@ -393,6 +393,11 @@ pub struct ChannelMonitor {
 	// the full block_connected).
 	pub(crate) last_block_hash: Sha256dHash,
 	secp_ctx: Secp256k1<secp256k1::All>, //TODO: dedup this a bit...
+
+	//this is all channels transactions that failed in the data_loss_protect protocol
+	//The first index is the height that we attempt to claim it at, this claimed height is by default one week later, and it is there to claim if the other party did not claim
+	transactions_awaiting_claiming : Vec<u32>,
```

Comment:
Wait, I thought we were gonna replace this with an broadcast_latest_local_at_height: Option<u32> as discussed at https://github.com/rust-bitcoin/rust-lightning/pull/244#discussion_r247694793

Reply:
Thats the height. 
Does the channel-monitor allow multiple channels to use it?